zict/file.py

```diff
@@ -68,7 +78,11 @@ def __getitem__(self, key):
 
     def __setitem__(self, key, value):
         with open(os.path.join(self.directory, _safe_key(key)), 'wb') as f:
-            f.write(value)
+            if isinstance(value, (tuple, list)):
+                for v in value:
+                    f.write(v)
```

Comment:
You can probably write `f.writelines(value)` instead, though I don't think that will bring much speedup.